src/app/components/MainContent/index.jsx

```diff
@@ -1,29 +1,19 @@
 import React from 'react';
-import styled from 'styled-components';
 import { any, arrayOf, shape } from 'prop-types';
-import { C_EBON, FF_NEWS_SANS_REG } from '../../../lib/constants/styles';
-
-const Headline = styled.h1`
-  color: ${C_EBON};
-  font-family: ${FF_NEWS_SANS_REG};
-  font-size: 2em;
-`;
+import Headline from '../Headline';
+import Text from '../Text';
 
 const render = ({ blocks }) =>
   blocks.map(block => {
     const { type, blockId } = block;
-    const blockString = JSON.stringify(block);
```

Comment:
Issue says "It should fall back to rendering a stringified block to a paragraph." so this shouldn't be removed entirely or the issue should be discussed and revised.